src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -1821,6 +1823,68 @@ export class ResultsViewPageStore {
         }
     });
 
+    readonly expressionProfiles = remoteData({
+        await:()=>[
+            this.molecularProfilesInStudies
+        ],
+        invoke:()=>{
+            return Promise.resolve(this.molecularProfilesInStudies.result.filter(
+                (profile:MolecularProfile)=>isExpressionProfile(profile.molecularProfileId, this.expressionTabseqVersion)
+            ));
+        }
+    });
+
+    readonly rnaSeqMolecularData = remoteData<{[hugoGeneSymbol:string]:NumericGeneMolecularData[][]}>({
+       await:()=>[
+           this.expressionProfiles,
+           this.genes,
+           this.geneMolecularDataCache
```

Comment:
can you use `numericGeneMolecularDataCache` for this instead?

Reply:
i think @adamabeshouse and i determined that both our cache strategies (new and old) are not better than simple client (request) level cache, which would be much simpler.  so will follow up with that after merge.